pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -323,12 +323,13 @@ func (c *Cluster) handleExternalMonStatus(status client.MonStatusResponse) error
 
 func (c *Cluster) addOrRemoveExternalMonitor(status client.MonStatusResponse) (bool, error) {
 	var changed bool
-	// clearing the content of clusterinfo monitors, it'll get populated in the next iteration
-	for mon := range c.ClusterInfo.Monitors {
-		delete(c.ClusterInfo.Monitors, mon)
+	oldClusterInfoMonitors := map[string]*cephconfig.MonInfo{}
+	// clearing the content of clusterinfo monitors, later it'll get populated again
+	for monName, mon := range c.ClusterInfo.Monitors {
+		oldClusterInfoMonitors[mon.Name] = mon
+		delete(c.ClusterInfo.Monitors, monName)
 	}
 	logger.Infof("ClusterInfo is now Empty, refilling it from status.MonMap.Mons")
```

Comment:
Come to think of it, this is clearly a Debug level log.

Reply:
It is fore sure.